### DISCUSSION TITLE
IKEA E1743 remote doesn't show any action after firmware upgrade to version 24.4.6

### DISCUSSION THREAD

**Discussion by Jaapop8** [Koenkk/zigbee2mqtt#18484] [Devices]
Hey, 
my device now shows only  battery and and link quality. But no action (it says N/A)
I tried all kinds of everything, but no result. 
Can anybody help? Thx in advance...

**Comment by ryancasler** [comment]
I have the same problem.  Did you ever get this resolved?

**Reply by Jaapop8** [reply]
Yes! I thought that one should see the possible actions, but it isn't....
Try to use the following Blueprint in Automations and it will work:
Zigbee2MQTT - IKEA TRADFRI - 5 Button Remote - Custom Actions
Good luck!

**Reply by Jaapop8** [reply]
Oops! I just assumed that you're using Home Assistant.... If you're not, I'll try to explain what the Blueprint does, so that you can implement it in your own way.

**Reply by ryancasler** [reply]
I'm not using home assistant.  But I don't see any states bein published at all by the remote inside of Zigbee2MQTT.  So, I'm a little confused.

**Comment by ryancasler** [comment]
Also, you had the 5 button remote?  I have the on-off remote.

**Reply by Jaapop8** [reply]
I also have IKEA on-off remotes working. And I don't see states either :) 
How are you using Z2M? I can explain in YAML what you should do, does that work for you?
